src/bindings/python/pyhirte/gen/agent.py

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+from typing import Callable
+from dasbus.typing import *
```

Comment:
I am not sure if it's required import * for dbus.typing, it's not generated api.

Reply:
Fair point - its not required, I was just lazy. I changed this to explicitly import the types. 
(btw. this is a generated file, the "original" was in the agent.tmpl)